.github/renovate.json5

```diff
@@ -0,0 +1,38 @@
+{
+  $schema: "https://docs.renovatebot.com/renovate-schema.json",
+  extends: ["config:recommended"],
+  schedule: ["* 19-21 * * 1"],
```

Comment:
We considered 1 week necessary with dependabot since it wasn't able to group stuff and we were afraid we might only build up outdated dependencies with larger intervals, but with grouping, I think we could use larger intervals.

Would it be possible to have 1 PR per month per group, and one every 3 months for the nix dependencies? If yes, @Kakadus @niklasmohrin @janno42 what would you think about that? Fewer commits in main -> less noise in git log.

Reply:
definitely possible and I see the rationale

Reply:
The intervals proposed by @richardebeling make sense to me